cypress/e2e/comments.spec.ts

```diff
@@ -175,46 +228,129 @@ describe("Comments", () => {
+        api
+          .createReviewRequest("test title", [E2E_EMAIL_COLLAB.email])
+          .then((id) => {
+            reviewId = id
+            cy.setEmailSessionDefaults("Email collaborator")
+            api.approveReviewRequest(reviewId).then((response) => {
+              cy.setEmailSessionDefaults("Email admin")
+              api.mergeReviewRequest(reviewId)
+            })
+          })
       })
-      it("should not be able to see comments for closed review requests", () => {
+      it("should not be able to see/create comments for merged review requests", () => {
         // Arrange
         // Act
+        cy.visit(
+          `${CMS_BASEURL}/sites/${E2E_EMAIL_TEST_SITE.repo}/review/${reviewId}`
+        )
+
         // Assert
+        checkCommentDisabled()
       })
-      it("should not be able to create comments for merged review requests", () => {
-        // Arrange
-        // Act
-        // Assert
+    })
+
+    describe("closed review requests", () => {
+      let reviewId: number
+      before(() => {
+        cy.intercept("GET", "**/comments").as(COMMENT_INTERCEPTOR)
+        // NOTE: Need to set permissions as order is `before` -> `beforeEach`
+        cy.setEmailSessionDefaults("Email admin")
+        cy.setupDefaultInterceptors()
+        api.closeReviewRequests()
+        cy.createEmailUser(
+          E2E_EMAIL_COLLAB.email,
+          "Email admin",
+          "Email admin",
+          E2E_EMAIL_TEST_SITE.name
+        )
+        api.editUnlinkedPage("faq.md", "some content", E2E_EMAIL_TEST_SITE.repo)
+        api
+          .createReviewRequest("test title", [E2E_EMAIL_COLLAB.email])
+          .then((id) => {
+            reviewId = id
+            cy.setEmailSessionDefaults("Email collaborator")
+            api.approveReviewRequest(reviewId).then((response) => {
+              cy.setEmailSessionDefaults("Email admin")
+              api.mergeReviewRequest(reviewId)
+            })
+          })
       })
-      it("should not be able to create comments for closed review requests", () => {
+      it("should not be able to see/create comments for closed review requests", () => {
         // Arrange
         // Act
+        cy.visit(
+          `${CMS_BASEURL}/sites/${E2E_EMAIL_TEST_SITE.repo}/review/${reviewId}`
+        )
         // Assert
+        checkCommentDisabled()
       })
-      it("should not be able to create comments for a non-existent review request", () => {
+    })
+
+    describe("non-existent review requests", () => {
+      before(() => {
+        cy.setEmailSessionDefaults("Email admin")
+        api.closeReviewRequests()
+      })
+      it("should not be able to see/create comments for non-existent review requests", () => {
         // Arrange
+        visitE2eEmailTestRepo()
+
         // Act
+
         // Assert
+        cy.contains(
+          `a[href^="/sites/${E2E_EMAIL_TEST_SITE.repo}/review/"]`
+        ).should("not.exist")
```

Comment:
at the point of testing this step, are we not still at the e2e test repo (url is `/dashboard`) and not the actual `/review/<id>` url?

Reply:
For this test there is no review requests, so it goes to the site dashboard, then checks if there is a link to the currently opened review request. This exerts that the link does not exist.

Reply:
ah ok this sounds logical - could we add this description above?